src/train.jl

```diff
@@ -3,6 +3,8 @@ const DEFAULT_DATA = Iterators.cycle((NullData(),))
 Base.iterate(::NullData, i=1) = nothing
 Base.length(::NullData) = 0
 
+@deprecate sciml_train(loss, θ, opt, args...; kwargs...) GalacticOptim.solve(optprob, opt)

```

Comment:
this deprecation is going to fail since `optprob` is not defined.

Reply:
Not sure how this deprecation should be handled actually. Can you be more explicit?

Reply:
Make it be a function call that builds the standard GalacticOptim problem and calls solve on it, but in there throws a warning that the function is deprecated. I think it's slightly too complicated for the standard `@deprecate` mechanism, though if you can shove it all into one call for that it would be nice.

Reply:
Hmm, can we just remove `sciml_train` now? (Easiest way I guess, otherwise I'll try the above)

Reply:
No.